hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -426,6 +430,20 @@ public static long getUtcTime() {
     return Calendar.getInstance(UTC_ZONE).getTimeInMillis();
   }
 
+  public static Policy getHttpPolicy(Configuration conf) {
+    String policyStr = conf.get("ozone.http.policy", OzoneConfigKeys.OZONE_HTTP_POLICY);
+    if(policyStr == null || policyStr.length() == 0) {
+      policyStr = conf.get("dfs.http.policy", DFSConfigKeys.DFS_HTTP_POLICY_DEFAULT);
+    }
```

Comment:
If HTTP_ONLY is used as the default, 
we can conf.get("OzoneConfigKeys.OZONE_HTTP_POLICY", DFSConfigKeys.DFS_HTTP_POLICY_DEFAULT);
Then we don't need checks of null, and we can make this logic simple.